docs/doc/30-reference/20-functions/123-file-functions/presign.md

```diff
@@ -0,0 +1,28 @@
+---
+title: PRESIGN
+---
+
+Generates the pre-signed URL for a staged file by the stage name and file path you provide. The pre-signed URL enables you to access the file through a web browswer or an API request.
+
+:::tip
+This function currently works with files stored at Amazon S3 only.
+:::
+
+## Syntax
+
+```sql
+PRESIGN @<stage_name>/.../<file_name>
```

Comment:
The full syntax should be:

```sql
PRESIGN [ { DOWNLOAD | UPLOAD }] @<stage_name>/.../<file_name> [ EXPIRE = <expire_in_seconds> ]
```

Reply:
What's the default value for EXPIRE? 3,600 seconds?